webui/react/src/services/types.ts

```diff
@@ -79,7 +81,7 @@ export interface PatchExperimentParams {
 
 export interface PatchExperimentState {
   experimentId: number;
-  state: RunState;
+  state: Determinedexperimentv1State;
```

Comment:
in general we want to avoid exposing types from the swagger bindings but I think we've made an exception for enums and constants right? we do the same with experiments endpoint right?

Reply:
I'm not sure, like told before is my first experience in a strong TS project (which I love), but if there's anything that can be improved just lmk, because there's an high chance I'm simply not seeing a very simple path :)

Reply:
**question:** do we still need `interface PatchExperimentState`? I don't see anything referencing it anymore in this PR.

Reply:
Removed the `PatchExperimentState` interface.